### PR TITLE
Temp dir issues on Windows

### PR DESCRIPTION
**Pull request by krishnaglick**
Node likes to do things for us!

### CODE REVIEW

File: config/api.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ exports.test = {
         'otherRoom': {},
       },
       paths: {
-        'locale': [ '/tmp/locale' ]
+        'locale': [ require('os').tmpdir() ]
```
Comment (evantahler) [inline config/api.js]:
```js
var path = require('path');
require('os').tmpdir()  + path.sep + 'locale'
```

Comment (evantahler) [inline config/api.js]: This would just be `/tmp` right now on *nix/OSX

Comment (evantahler) [inline config/api.js]: And that's why the tests are failing.

**Comment by krishnaglick** [comment]
Works on windows now.

**Comment by evantahler** [comment]
and linux!

**Comment by evantahler** [comment]
fyi: OSX does something wacky with tmp paths: https://github.com/evantahler/actionhero/commit/6868196338dbf0038ac008794bcb8f0be1ee6bce

**Comment by evantahler** [comment]
hey @krishnaglick ... how does the /test/server/web.js tests work if there's no `/tmp` on windows?

**Comment by krishnaglick** [comment]
On windows the temp folder is located at '%appdata%/Local/Temp'
This roughly translates to 'driveLetter:\Users\accountName\AppData\Local\Temp'

**Comment by evantahler** [comment]
sure... so that test wan't running either then?

**Comment by krishnaglick** [comment]
Huh? An AH test?

**Pull request merged**